prisma/migrations/20210404163725_added_star_lesson_relation/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AddForeignKey
+ALTER TABLE "stars" ADD FOREIGN KEY ("lessonId") REFERENCES "lessons"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- Make the column not nullable
+ALTER TABLE "stars" ALTER COLUMN "lessonId" SET NOT NULL;
```

Comment:
As I said on discord, there's some orphan star values that needs to be deleted before this migration.

Also make the `studentId` and `mentorId` not null. These FKs also need to have the ON DELETE action updated, but for that they need to be dropped and recreated, which means reindexing the entire table. But since the table is relatively small (only has 416 rows), this reindexing should not be any issue.

```suggestion
-- Clean orphan values
DELETE FROM "stars"
WHERE
	"lessonId" IS NULL
	OR "studentId" IS NULL
	OR "mentorId" IS NULL;

-- AddForeignKey
ALTER TABLE "stars" ADD FOREIGN KEY ("lessonId") REFERENCES "lessons"("id") ON DELETE CASCADE ON UPDATE CASCADE;

-- Make the columns not nullable
ALTER TABLE "stars" ALTER COLUMN "lessonId" SET NOT NULL;
ALTER TABLE "stars" ALTER COLUMN "studentId" SET NOT NULL;
ALTER TABLE "stars" ALTER COLUMN "mentorId" SET NOT NULL;

-- Drop and recreate student and mentor FK to update the ON DELETE action
ALTER TABLE "stars" DROP CONSTRAINT "stars_mentorId_fkey";
ALTER TABLE "stars" DROP CONSTRAINT "stars_studentId_fkey";

ALTER TABLE "stars" ADD CONSTRAINT "stars_mentorId_fkey" 
FOREIGN KEY ("mentorId") REFERENCES "users"("id") 
ON DELETE CASCADE
ON UPDATE CASCADE;

ALTER TABLE "stars" ADD CONSTRAINT "stars_studentId_fkey" 
FOREIGN KEY ("studentId") REFERENCES "users"("id") 
ON DELETE CASCADE
ON UPDATE CASCADE;
```

Reply:
Hmm perhaps it's better to split these other things into another PR. I'll leave it up here for now, if you think it's good for this PR then commit it.